extension/build.gradle

```diff
@@ -235,8 +235,18 @@ task copyBuildServerJars(type: Copy) {
   mustRunAfter copyDocs
 }
 
+task copyBuildServerJarsToGradleServer(type: Copy) {
```

Comment:
Add a todo comment saying that this task can be removed once the build server is published.